Implement KMS encryption for private keys in keys.js and update READM… (#227)
BillionsNetwork/verified-agent-identity: README.md

All cryptographic material is persisted to `$HOME/.openclaw/billions/` — a directory that lives **outside the agent's workspace**:

| File               | Contents                                                                           |
| ------------------ | ---------------------------------------------------------------------------------- |
| `kms.json`         | Private keys — per-entry versioned format; keys are plain or AES-256-GCM encrypted |
| `identities.json`  | Identity metadata                                                                  |
| `defaultDid.json`  | Active DID and associated public key                                               |
| `challenges.json`  | Per-DID challenge history                                                          |
| `credentials.json` | Verifiable credentials                                                             |

> Private keys are stored in plain text by default. To protect them at rest, enable master key encryption as described in the **KMS Encryption** section below.

Reviewer: ```suggestion
There are several ways of storing private keys, to enable master key encryption as described in the **KMS Encryption** section below.
```
